### DISCUSSION TITLE
Sliders increment

### DISCUSSION THREAD

**Discussion by ArcadiaBaes** [Mustard2/MustardUI#195] [Help]
I made a texture selector node and added it to the UI. It works, however the increment is too big (or too small i dont know). I would like it to move one by one, and not like in my screenshoot. I cant find where to edit it. 

![image](https://github.com/user-attachments/assets/d3f3c723-ad72-4757-b120-34e8646d1726)

**Comment by ArcadiaBaes** [comment]
![image](https://github.com/user-attachments/assets/4d52496b-12c6-4260-b36b-e03fa81426a4)
i think i have to enable this force type, but when i do it i have this message
![image](https://github.com/user-attachments/assets/6ef452c5-ea54-435f-82f5-bc9b30a074fc)

**Comment by Mustard2** [comment]
Ok this seems like a bug, it shoudn't happen. Do you have a blend file I can check where this error is appearing?

**Comment by ArcadiaBaes** [comment]
i can send you a blend file but i guess the issue comes from ops_menu_settings.py

**Reply by Mustard2** [reply]
Yeah it's that I can not replicate it on my side, maybe there's a corner case I didn't consider

**Comment by Mustard2** [comment]
Opened an issue to keep track of this: https://github.com/Mustard2/MustardUI/issues/200

Closing this, please open the linked Issue for further discussion
